program/src/snapshot_vault_operator_delegation.rs

```diff
@@ -111,13 +109,13 @@ pub fn process_snapshot_vault_operator_delegation(
     };
 
     let (ncn_fee_group, reward_multiplier_bps) = {
-        let tracked_mints_data = tracked_mints.data.borrow();
-        let tracked_mints_account = TrackedMints::try_from_slice_unchecked(&tracked_mints_data)?;
-        let mint_entry = tracked_mints_account.get_mint_entry(vault_index)?;
+        let weight_table_data = weight_table.data.borrow();
+        let weight_table_account = WeightTable::try_from_slice_unchecked(&weight_table_data)?;
+        let weight_entry = weight_table_account.get_weight_entry(&st_mint)?;
 
         (
-            mint_entry.ncn_fee_group(),
-            mint_entry.reward_multiplier_bps(),
+            weight_entry.mint_entry().ncn_fee_group(),
+            weight_entry.mint_entry().reward_multiplier_bps(),
```

Comment:
weight table account is also deserialized below, should consolidate so we only deser once

Reply:
Done